src/drivers/sifive_ccache0.c

```diff
@@ -56,21 +50,20 @@ void sifive_ccache0_get_config(sifive_ccache0_config *config) {
         val = REGW(METAL_SIFIVE_CCACHE0_CONFIG);
 
         /* Populate cache configuration data */
-        config->num_bank = (val & SIFIVE_CCACHE0_CONFIG_BANKS_MASK);
-        config->num_ways =
-            (val & SIFIVE_CCACHE0_CONFIG_WAYS_MASK) >> REG_SHIFT_8;
+        config->num_bank = (val & SIFIVE_CCACHE0_BYTE_MASK);
```

Comment:
Adding a comment describing what being done here would help the reader, 
given that there is a lot bit manipulation (shifting and anding)

Reply:
added a comment, describing the operation.